### DISCUSSION TITLE
[DE] [Suggestion] | Anfrage auf TS3 Plugin

### DISCUSSION THREAD

**Discussion by SnakeEater96** [LeonMarcel-HD/StreamDeck-TeamSpeak5#6] [Feature Requests / Suggestions]
Moin,
der User der das Streamdeck Plugin für TS3 gebaut hat baut an dem Projekt nicht mehr weiter und hat es schon Archiviert nun meine frage an dich könntest du das Plugin ggf. weiter führen und ergänzen?
mir fehlt da Plugins zuAktivieren und Deaktivieren
https://github.com/ZerGo0/streamdeck-teamspeak3integration

**Comment by LeonMarcel-HD** [comment]
Ich werde für TeamSpeak 3 kein Plugin entwickeln, da es keine Zukunft mehr hat und die Entwicklung mehr oder weniger schon abgeschlossen ist und sich das TeamSpeak team auf TeamSpeak 5 konzentriert.

Wobei ich ich mich geehrt fühle, dass du da an mich denkst, möchte ich auch garnicht das Plugin weiter führen, ZerGo0 hat das Projekt als abgeschlossen gesehen und bräuchte meiner Meinung nach einen kompletten Neubau um meinen Anforderungen folge zu leisten. Zudem welche Funktion fehlt dir in dem alten Plugin? Meinst du die Drücken zum aktivieren funktionen bei den whisper / push to talk buttons?
